setup.py

```diff
@@ -16,20 +16,15 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from setuptools import setup
+from distutils.util import convert_path
 
-import imp
+version = {}
+exec(open(convert_path('pycoast/version.py')).read(), version)
```

Comment:
Or another option that I don't like:

4. Store version in setup.py and then query it inside the package via setuptools to get `pycoast.__version__`.